caching-service/src/main/resources/caching-log-messages.yml

```diff
@@ -57,6 +57,20 @@ messages:
         reason: "The storage space is full."
         action: "Change the definition of the VSAM and the configuration for the maximum records stored or replace Reject strategy with Remove Oldest."
 
+    -   key: org.zowe.apiml.cache.duplicateValue
+        number: ZWECS135
+        type: ERROR
+        text: "Adding value '%s' resulted in a collision in the cache."
+        reason: "Value is already in the cache."
+        action: "Provide a value that is in the cache."
```

Comment:
Typo: 
Required action is probably: "Provide a value that is **not** in the cache."

Reply:
Good catch, thanks Petr, I will fix it